pkg/systemd/host.js

```diff
@@ -663,9 +664,44 @@ PageSystemInformationChangeSystime.prototype = {
         $('#systime-timezones').on('change', enable_apply_button);
         $('#systime-date-input').on('focusin', $.proxy(this, "store_date"));
         $('#systime-date-input').on('focusout', $.proxy(this, "restore_date"));
+
+        this.ntp_servers_tmpl = $("#ntp-servers-tmpl").html();
```

Comment:
Shouldn't we use 'self' here, and on the lines below? Especially in big functions, it makes the code much easier to reason about.